### ISSUE TITLE
[V3 Downloader] Repo add error on bad cog

### ISSUE THREAD

**Issue by bobloy** [Cog-Creators/Red-DiscordBot#1654]
# Command bugs

#### Command name

`repo add`

#### What cog is this command from?

Downloader

#### What were you expecting to happen?

Properly add a repo

#### What actually happened?

Repo entered an error state where files were downloaded but repo was not added

#### How can we reproduce this issue?

`repo add` on a repo with a cog that can't be imported due to an error (such as syntax)

**Comment by tekulvw** [comment]
Which repo and where's the traceback?

**Comment by bobloy** [comment]
```[14/05/2018 16:19] ERROR events on_command_error 206: Exception in command 'repo add'
Traceback (most recent call last):
  File "C:\Users\Bobloy\Documents\Fox-V3\lib\site-packages\discord\ext\commands\core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "C:\Users\Bobloy\Documents\Fox-V3\lib\site-packages\redbot\cogs\downloader\downloader.py", line 229, in _repo_add
    branch=branch
  File "C:\Users\Bobloy\Documents\Fox-V3\lib\site-packages\redbot\cogs\downloader\repo_manager.py", line 567, in add_repo
    await r.clone()
  File "C:\Users\Bobloy\Documents\Fox-V3\lib\site-packages\redbot\cogs\downloader\repo_manager.py", line 207, in clone
    return self._update_available_modules()
  File "C:\Users\Bobloy\Documents\Fox-V3\lib\site-packages\redbot\cogs\downloader\repo_manager.py", line 149, in _update_available_modules
    for file_finder, name, is_pkg in pkgutil.walk_packages(path=[str(self.folder_path), ]):
  File "C:\Users\Bobloy\AppData\Local\Programs\Python\Python36-32\lib\pkgutil.py", line 92, in walk_packages
    __import__(info.name)
  File "C:\Users\Bobloy\Documents\GitHub\Fox-V3\fight\__init__.py", line 1, in <module>
    from .fight import Fight
  File "C:\Users\Bobloy\Documents\GitHub\Fox-V3\fight\fight.py", line 25, in <module>
    class Fight:
  File "C:\Users\Bobloy\Documents\GitHub\Fox-V3\fight\fight.py", line 94, in Fight
    @commands.guild_only()
AttributeError: module 'redbot.core.commands.commands' has no attribute 'guild_only'
```

**Comment by bobloy** [comment]
There is an error in the `fight` cog that prevents it from being imported, and this crashes downloader during `repo add`

**Comment by tekulvw** [comment]
Fixed with #1655
